### PR TITLE
[Core Branding] Font Fixes (Web / Mobile)

### PR DESCRIPTION
**Pull request by grgia**
### Details

Fixes android fonts, and updates font-family for crispier fonts. Also fixes italic mono console error on IOS. Note that with our new fonts, we do not support mono italics on mobile.

IOS error: 
image

For future reference, [this article](https://mehrankhandev.medium.com/ultimate-guide-to-use-custom-fonts-in-react-native-77fcdf859cf4) was helpful.

### Fixed Issues

$ https://github.com/Expensify/App/issues/13413

### Tests

- [ ] Verify that no errors appear in the JS console
- [ ] Open a chat and send a bold, italic, italic and bold, strikethrough, and regular chat
- [ ] Do the same for mono 
- [ ] Open the console and verify that CSS for each font is using `ExpensifyNeue-Regular`

### QA Steps

- [ ] Verify that no errors appear in the JS console
- [ ] Open a chat and send a ``` *bold*, _italic_, *_italic and bold_*, ~strikethrough~, and regular chat ```
- [ ] Do the same for  ``` `mono` (ex: *`mono bold`*) ```
- [ ] Open the console and verify that CSS for each font is using `ExpensifyNeue-Regular`

### PR Author Checklist

- [x] I linked the correct issue in the `### Fixed Issues` section above
- [x] I wrote clear testing steps that cover the changes made in this PR
    - [x] I added steps for local testing in the `Tests` section
    - [x] I added steps for the expected offline behavior in the `Offline steps` section
    - [x] I added steps for Staging and/or Production testing in the `QA steps` section
    - [x] I added steps to cover failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
    - [x] I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
    - [x] I tested this PR with a [High Traffic account](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#high-traffic-accounts) against the staging or production API to ensure there are no regressions (e.g. long loading states that impact usability).
- [x] I included screenshots or videos for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [x] I ran the tests on **all platforms** & verified they passed on:
    - [x] iOS / native
    - [x] Android / native
    - [x] iOS / Safari
    - [x] Android / Chrome
    - [x] MacOS / Chrome
    - [x] MacOS / Desktop
- [x] I verified there are no console errors (if there's a console error not related to the PR, report it or open an issue for it to be fixed)
- [x] I followed proper code patterns (see [Reviewing the code](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [x] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. `toggleReport` and not `onIconClick`)
    - [x] I verified that comments were added to code that is not self explanatory
    - [x] I verified that any new or modified comments were clear, correct English, and explained "why" the code was doing something instead of only explaining "what" the code was doing.
    - [x] I verified any copy / text shown in the product is localized by adding it to `src/languages/*` files and using the [translation method](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60)
    - [x] I verified all numbers, amounts, dates and phone numbers shown in the product are using the [localization methods](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60-L68)
    - [x] I verified any copy / text that was added to the app is correct English and approved by marketing by adding the `Waiting for Copy` label for a copy review on the original GH to get the correct copy.
    - [x] I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named "index.js". All platform-specific files are named for the platform the code supports as outlined in the README.
    - [x] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#jsdocs)) were followed
- [x] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
- [x] I followed the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md)
- [x] I tested other components that can be impacted by my changes (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` are working as expected)
- [x] I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)
- [x] I verified any variables that can be defined as constants (ie. in CONST.js or at the top of the file that uses the constant) are defined as such
- [x] I verified that if a function's arguments changed that all usages have also been updated correctly
- [x] If a new component is created I verified that:
    - [x] A similar component doesn't exist in the codebase
    - [x] All props are defined accurately and each prop has a `/** comment above it */`
    - [x] The file is named correctly
    - [x] The component has a clear name that is non-ambiguous and the purpose of the component can be inferred from the name alone
    - [x] The only data being stored in the state is data necessary for rendering and nothing else
    - [x] For Class Components, any internal methods passed to components event handlers are bound to `this` properly so there are no scoping issues (i.e. for `onClick={this.submit}` the method `this.submit` should be bound to `this` in the constructor)
    - [x] Any internal methods bound to `this` are necessary to be bound (i.e. avoid `this.submit = this.submit.bind(this);` if `this.submit` is never passed to a component event handler like `onClick`)
    - [x] All JSX used for rendering exists in the render method
    - [x] The component has the minimum amount of code necessary for its purpose, and it is broken down into smaller components in order to separate concerns and functions
- [x] If any new file was added I verified that:
    - [x] The file has a description of what it does and/or why is needed at the top of the file if the code is not self explanatory
- [x] If a new CSS style is added I verified that:
    - [x] A similar style doesn't already exist
    - [x] The style can't be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/styles/StyleUtils.js) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(themeColors.componentBG`)
- [x] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
- [x] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.
- [x] If a new page is added, I verified it's using the `ScrollView` component to make it scrollable when more elements are added to the page.
- [x] I have checked off every checkbox in the PR author checklist, including those that don't apply to this PR.

### Screenshots/Videos
<details>
<summary>Web</summary>

image
Screenshot 2022-12-07 at 2 55 26 PM

image

</details>

<details>
<summary>Mobile Web - Chrome</summary>

image

</details>

<details>
<summary>Mobile Web - Safari</summary>

image
image

image

</details>

<details>
<summary>Desktop</summary>

image
image

image

</details>

<details>
<summary>iOS</summary>

image
image

image

</details>

<details>
<summary>Android</summary>

image
image

image
image

</details>

### CODE REVIEW

File: assets/css/fonts.css
Comment (shawnborton) [inline assets/css/fonts.css:37]: So for whatever reason, it looks like the CSS is actually calling `ExpensifyMono-Bold` when we are using bold mono. So technically this is working fine now. However, we could update the font family name and then make updates elsewhere if we want to, it would be cleaner that way. 

image

Comment (grgia) [reply assets/css/fonts.css:37]: Looks like we use a helper function to get the correct mono fonts, so I'll revert this one for now. 

https://github.com/Expensify/App/blob/master/src/styles/StyleUtils.js#L329-#L335

**Comment by shawnborton** [comment]
Did some local testing and this is feeling pretty good to me! 
image

Would be curious to see this on mobile web to see if we solved it over there too.

**Comment by grgia** [comment]
Added screenshots for mobile web on emulators

**Comment by grgia** [comment]
Waiting for android build for final screenshot, so I'm gonna open this for review now

**Comment by grgia** [comment]
It looks like mono styles don't work on android (on main as well). I posted screenshots

**Comment by grgia** [comment]
I'm not thinking, this PR is for web styles. Do we want to allow styled mono on android/ios? Or do nothing and leave it as is for now

**Comment by shawnborton** [comment]
We should definitely plan to fix mono - can you share screenshots of what that looks like? Also while you are at it, could you confirm that the new font is working on the native Android app?

In terms of fixing mono though, we could totally do it in a separate PR, or bundle it into this one if you'd like to - your call!

**Comment by grgia** [comment]
Here's the screenshot I took before on native android- looks like the mono styles are all the same
image

**Comment by grgia** [comment]
https://expensify.slack.com/archives/C01GTK53T8Q/p1670546775495159
image

sike ...looks like I broke it 😂. Gonna throw this back in WIP to fix

**Comment by shawnborton** [comment]
Oh wow, but it looks like none of the regular Neue fonts are working correctly on Android?

**Comment by grgia** [comment]
Would someone mind building this branch on android and sending a screenshot?

**Comment by mananjadhav** [comment]
@grgia do you need an Android physical device? or emulator is fine here?

**Comment by grgia** [comment]
If you are set up to test DEV on a physical android, that would be great. Otherwise, the emulator should be fine.

**Comment by grgia** [comment]
This is ready for review! Here's updated screenshots.
@shawnborton I believe if we want to support italic mono on mobile, we may need to add the font files?
| Mobile Chrome  | Android | Mobile Safari | IOS|
|----|----|----|----|
|image|image|image|image|

| Web | Desktop |
|---|---|
| image|image|

**Comment by mananjadhav** [comment]
@grgia This is how it looks on my physical Android device. The `mono with 1` looks a bit different in your screenshot vs mine.

android-fonts

-------

> we want to support italic mono 

Based on the answer to this I guess our values `MONOSPACE_ITALIC` and `MONOSPACE_BOLD_ITALIC` will update right?

**Comment by grgia** [comment]
@mananjadhav That looks good! `the mono with a 1` on mine is actually `mono with an l` (L), I tested with numbers and that looks correct to me

**Comment by mananjadhav** [comment]
Thanks for clarifying @grgia. Testing out the other platforms now.

**Comment by shawnborton** [comment]
> I believe if we want to support italic mono on mobile, we may need to add the font files?

@grgia yup, I say we can follow up with that one as we'll first need to have the foundry (CoType) create these files/styles for us.

**Review by shawnborton — DISMISSED** [review]

**Comment by mananjadhav** [comment]
## Reviewer Checklist

- [x] I have verified the author checklist is complete (all boxes are checked off).
- [x] I verified the correct issue is linked in the `### Fixed Issues` section above
- [x] I verified testing steps are clear and they cover the changes made in this PR
    - [x] I verified the steps for local testing are in the `Tests` section
    - [x] I verified the steps for Staging and/or Production testing are in the `QA steps` section
    - [x] I verified the steps cover any possible failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
    - [x] I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
- [x] I checked that screenshots or videos are included for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [x] I included screenshots or videos for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [x] I verified tests pass on **all platforms** & I tested again on:
    - [x] iOS / native
    - [x] Android / native
    - [x] iOS / Safari
    - [x] Android / Chrome
    - [x] MacOS / Chrome
    - [x] MacOS / Desktop
- [x] If there are any errors in the console that are unrelated to this PR, I either fixed them (preferred) or linked to where I reported them in Slack
- [x] I verified proper code patterns were followed (see [Reviewing the code](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [x] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. `toggleReport` and not `onIconClick`).
    - [x] I verified that comments were added to code that is not self explanatory
    - [x] I verified that any new or modified comments were clear, correct English, and explained "why" the code was doing something instead of only explaining "what" the code was doing.
    - [x] I verified any copy / text shown in the product is localized by adding it to `src/languages/*` files and using the [translation method](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60)
    - [x] I verified all numbers, amounts, dates and phone numbers shown in the product are using the [localization methods](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60-L68)
    - [x] I verified any copy / text that was added to the app is correct English and approved by marketing by adding the `Waiting for Copy` label for a copy review on the original GH to get the correct copy.
    - [x] I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named "index.js". All platform-specific files are named for the platform the code supports as outlined in the README.
    - [x] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#jsdocs)) were followed
- [x] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
- [x] I verified that this PR follows the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md)
- [ ] I verified other components that can be impacted by these changes have been tested, and I retested again (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` have been tested & I retested again)
- [x] I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)
- [x] I verified any variables that can be defined as constants (ie. in CONST.js or at the top of the file that uses the constant) are defined as such
- [x] If a new component is created I verified that:
    - [x] A similar component doesn't exist in the codebase
    - [x] All props are defined accurately and each prop has a `/** comment above it */`
    - [x] The file is named correctly
    - [x] The component has a clear name that is non-ambiguous and the purpose of the component can be inferred from the name alone
    - [x] The only data being stored in the state is data necessary for rendering and nothing else
    - [x] For Class Components, any internal methods passed to components event handlers are bound to `this` properly so there are no scoping issues (i.e. for `onClick={this.submit}` the method `this.submit` should be bound to `this` in the constructor)
    - [x] Any internal methods bound to `this` are necessary to be bound (i.e. avoid `this.submit = this.submit.bind(this);` if `this.submit` is never passed to a component event handler like `onClick`)
    - [x] All JSX used for rendering exists in the render method
    - [x] The component has the minimum amount of code necessary for its purpose, and it is broken down into smaller components in order to separate concerns and functions
- [x] If any new file was added I verified that:
    - [x] The file has a description of what it does and/or why is needed at the top of the file if the code is not self explanatory
- [x] If a new CSS style is added I verified that:
    - [x] A similar style doesn't already exist
    - [x] The style can't be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/styles/StyleUtils.js) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(themeColors.componentBG`)
- [x] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
- [ ] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.
- [x] If a new page is added, I verified it's using the `ScrollView` component to make it scrollable when more elements are added to the page.
- [x] I have checked off every checkbox in the PR reviewer checklist, including those that don't apply to this PR.

### Screenshots/Videos
<details>
<summary>Web</summary>

web-font-styles

</details>

<details>
<summary>Mobile Web - Chrome</summary>

mweb-chrome-font-styles

</details>

<details>
<summary>Mobile Web - Safari</summary>

mweb-safari-font-styles

</details>

<details>
<summary>Desktop</summary>

desktop-font-styles

</details>

<details>
<summary>iOS</summary>

ios-font-styles

</details>

<details>
<summary>Android</summary>

android-font-styles

</details>

**Comment by mananjadhav** [comment]
@grgia do we maintain any internal regression test suite for fonts & styles? If yes, can that be shared here so that I can test that too. Otherwise rest of it looks good and tests well.

**Comment by shawnborton** [comment]
Screenshots look great, nice and crispy - nice work Georgia!

**Comment by grgia** [comment]
@mananjadhav we don't currently have regression test steps for fonts, but will be adding them as core branding changes are finalized.

**Comment by mananjadhav** [comment]
Okay thanks for the update. I’ve tested them the composer with all font styles. I’ve also completed the checklist.

**Review by roryabraham — CHANGES_REQUESTED** [review]
Okay, so I think we want to make some changes to this PR. Namely, the `.woff` and `.woff2` assets are being bundled in the iOS and Android apps, despite the fact that they are not used on that platform. This is something that I solved in [this PR](https://github.com/Expensify/App/pull/12983), and we can borrow part of that code for this PR

To fix this, I think we'll want to:

1. establish a directory structure like this:

	```
	assets/fonts/
	----------------|native  // all the .otf fonts for the mobile app
	----------------|web    // all the .woff2 and .woff fonts for the web app (plus .otf for NewKansas, for now)
	```

1. Make [this change](https://github.com/Expensify/App/pull/12983/files#diff-8be869fd4628b75fa5a2b47a309cc53741d78a47478966dc3e4b3c50ffce00d9R6) in react-native.config.js to use `assets/fonts/native` for native font assets.
1. Update [this line](https://github.com/Expensify/App/blob/12f56eafd6c6709eae59b3eaea052064be0c70c6/config/webpack/webpack.common.js#L93) to copy web fonts from `assets/fonts/web` instead of `assets/fonts`
1. Make sure we update [this line](https://github.com/Expensify/App/blob/12f56eafd6c6709eae59b3eaea052064be0c70c6/config/webpack/webpack.common.js#L199) to also look for the ExpensifyNewKansas `.otf` files on web.

**Comment by shawnborton** [comment]
New Kansas isn't actually ready to go yet, we haven't finalized the EULA - so that being said, it should have never been added to begin with. So if anything, I say we just rip out the NK files from this PR and then we follow up as a separate PR/issue to add NK when we get the green light.

**Comment by roryabraham** [comment]
Ok, so knowing that we can just:

- Replace NewKansas with the old header font (Fabriga, if memory serves?)
- Nix this change from my review above:

    > ~~Make sure we update [this line](https://github.com/Expensify/App/blob/12f56eafd6c6709eae59b3eaea052064be0c70c6/config/webpack/webpack.common.js#L199) to also look for the ExpensifyNewKansas .otf files on web.~~

**Comment by shawnborton** [comment]
We should definitely not include Fabriga, that is not a brand font. Let's just get rid of the NK files for now and keep all of the font styles as they are, where are just using Expensify Neue for headline moments.

**Comment by shawnborton** [comment]
Or yeah, like you said, just do nothing with the NK files that are there.

**Comment by grgia** [comment]
I just renamed the New Kansas file to match postscript in the android assets folder so when we do decide to use it, it won't break. Is it going to be replaced in the future anyways?

**Comment by shawnborton** [comment]
It won't be replaced, it's the actual font (and font files, I think) that we'll be using.

**Review by roryabraham — APPROVED** [review]
LGTM now, thanks for making those changes :bow:

**Comment by grgia** [comment]
Made changes and added new screenshots!

**Comment by roryabraham** [comment]
@mananjadhav Can you please retest and provide updated screenshots?

**Review by shawnborton — APPROVED** [review]

**Comment by mananjadhav** [comment]
## Reviewer Checklist

- [x] I have verified the author checklist is complete (all boxes are checked off).
- [x] I verified the correct issue is linked in the `### Fixed Issues` section above
- [x] I verified testing steps are clear and they cover the changes made in this PR
    - [x] I verified the steps for local testing are in the `Tests` section
    - [x] I verified the steps for Staging and/or Production testing are in the `QA steps` section
    - [x] I verified the steps cover any possible failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
    - [x] I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
- [x] I checked that screenshots or videos are included for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [x] I included screenshots or videos for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [ ] I verified tests pass on **all platforms** & I tested again on:
    - [x] iOS / native
    - [x] Android / native
    - [ ] iOS / Safari
    - [ ] Android / Chrome
    - [x] MacOS / Chrome
    - [x] MacOS / Desktop
- [x] If there are any errors in the console that are unrelated to this PR, I either fixed them (preferred) or linked to where I reported them in Slack
- [x] I verified proper code patterns were followed (see [Reviewing the code](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [x] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. `toggleReport` and not `onIconClick`).
    - [x] I verified that comments were added to code that is not self explanatory
    - [x] I verified that any new or modified comments were clear, correct English, and explained "why" the code was doing something instead of only explaining "what" the code was doing.
    - [x] I verified any copy / text shown in the product is localized by adding it to `src/languages/*` files and using the [translation method](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60)
    - [x] I verified all numbers, amounts, dates and phone numbers shown in the product are using the [localization methods](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60-L68)
    - [x] I verified any copy / text that was added to the app is correct English and approved by marketing by adding the `Waiting for Copy` label for a copy review on the original GH to get the correct copy.
    - [x] I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named "index.js". All platform-specific files are named for the platform the code supports as outlined in the README.
    - [x] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#jsdocs)) were followed
- [x] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
- [x] I verified that this PR follows the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md)
- [x] I verified other components that can be impacted by these changes have been tested, and I retested again (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` have been tested & I retested again)
- [x] I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)
- [x] I verified any variables that can be defined as constants (ie. in CONST.js or at the top of the file that uses the constant) are defined as such
- [x] If a new component is created I verified that:
    - [x] A similar component doesn't exist in the codebase
    - [x] All props are defined accurately and each prop has a `/** comment above it */`
    - [x] The file is named correctly
    - [x] The component has a clear name that is non-ambiguous and the purpose of the component can be inferred from the name alone
    - [x] The only data being stored in the state is data necessary for rendering and nothing else
    - [x] For Class Components, any internal methods passed to components event handlers are bound to `this` properly so there are no scoping issues (i.e. for `onClick={this.submit}` the method `this.submit` should be bound to `this` in the constructor)
    - [x] Any internal methods bound to `this` are necessary to be bound (i.e. avoid `this.submit = this.submit.bind(this);` if `this.submit` is never passed to a component event handler like `onClick`)
    - [x] All JSX used for rendering exists in the render method
    - [x] The component has the minimum amount of code necessary for its purpose, and it is broken down into smaller components in order to separate concerns and functions
- [x] If any new file was added I verified that:
    - [x] The file has a description of what it does and/or why is needed at the top of the file if the code is not self explanatory
- [x] If a new CSS style is added I verified that:
    - [x] A similar style doesn't already exist
    - [x] The style can't be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/styles/StyleUtils.js) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(themeColors.componentBG`)
- [x] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
- [x] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.
- [x] If a new page is added, I verified it's using the `ScrollView` component to make it scrollable when more elements are added to the page.
- [x] I have checked off every checkbox in the PR reviewer checklist, including those that don't apply to this PR.

### Screenshots/Videos
<details>
<summary>Web</summary>

web-font-styles

</details>

<details>
<summary>Mobile Web - Chrome</summary>

Pending


</details>

<details>
<summary>Mobile Web - Safari</summary>

Pending


</details>

<details>
<summary>Desktop</summary>

desktop-font-styles

</details>

<details>
<summary>iOS</summary>

ios-font-styles

</details>

<details>
<summary>Android</summary>

android-font-styles

</details>

--- 
@roryabraham @grgia  Tested again but unable to run this on mWeb - Chrome and Safari both due to the Auth errors.

File: src/styles/fontFamily/index.js
Comment (shawnborton) [inline src/styles/fontFamily/index.js:7]: Sorry for thinking of this so late... but I guess semantically, it makes more sense to just call this "Expensify Neue" or in this case, `'ExpensifyNeue'` as that's the name of the font family. "Regular" is simply a weight of the font, like Bold or Semi-Bold or Light, etc. No need to fix this immediately, but just wanted to jot my thoughts down.

Comment (grgia) [reply src/styles/fontFamily/index.js:7]: I agree! could we bundle this with the new kansas changes?

Comment (shawnborton) [reply src/styles/fontFamily/index.js:7]: I'm down with that@ cc @luacmartins as I saw you picked up the NK changes.

Comment (roryabraham) [reply src/styles/fontFamily/index.js:7]: Just FYI we created a separate issue for this change: https://github.com/Expensify/App/issues/13772

**Pull request merged**
